include/deal.II/base/ndarray.h

```diff
@@ -0,0 +1,102 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 1921 by the deal.II authors
```

Comment:
Are we going back in time ;)

Reply:
Ha, that's funny!

Reply:
https://www.youtube.com/watch?v=2YymGJKhGgY

Reply:
https://www.youtube.com/watch?v=m__wmsIn99E

Reply:
https://www.youtube.com/watch?v=AM5EYO5wWMA